modules/oc-mirror-command-reference.adoc

```diff
@@ -69,7 +69,7 @@ The following tables describe the `oc mirror` subcommands and flags:
 |Enable mirroring for local OCI catalogs on disk to the target mirror registry.
 
 |`--manifests-only`
-|Generate manifests for `ImageContentSourcePolicy` objects to configure a cluster to use the mirror registry, but do not actually mirror any images.
+|Generate manifests for `ImageContentSourcePolicy` objects to configure a cluster to use the mirror registry, but do not actually mirror any images. To use this flag, you must pass in an image set with the `--from` flag.
```

Comment:
My ignorance leading to a probably dumb question: 

Does this mean 
* pass in an **image set** by using the `--from` flag.
* pass in an **image** that is set with the `--from` flag.